test/fixtures/test-runner/output/test-timeout-flag.snapshot

```diff
@@ -1,16 +1,10 @@
 TAP version 13
 # Subtest: --test-timeout is set to 20ms
     # Subtest: should timeout after 20ms
-    not ok 1 - should timeout after 20ms
```

Comment:
I think this might be incorrect, the test should not pass if `--test-timeout=20` is passed as a CLI flag.

Reply:
Thanks for pointing that out! I took a quick look and realized I had misread this change. I’ll take another look at it 😁